pkg/ansible/proxy/inject_owner.go

```diff
@@ -126,21 +139,7 @@ func (i *injectOwnerReferenceHandler) ServeHTTP(w http.ResponseWriter, req *http
 			if addOwnerRef {
 				data.SetOwnerReferences(append(data.GetOwnerReferences(), owner.OwnerReference))
 			} else {
-				ownerGV, err := schema.ParseGroupVersion(owner.APIVersion)
-				if err != nil {
-					m := fmt.Sprintf("could not get group version for: %v", owner)
-					log.Error(err, m)
-					http.Error(w, m, http.StatusBadRequest)
-					return
-				}
-				a := data.GetAnnotations()
-				if a == nil {
-					a = map[string]string{}
-				}
-				a[osdkHandler.NamespacedNameAnnotation] = strings.Join([]string{owner.Namespace, owner.Name}, "/")
-				a[osdkHandler.TypeAnnotation] = fmt.Sprintf("%v.%v", owner.Kind, ownerGV.Group)
-
-				data.SetAnnotations(a)
+				handler.SetOwnerAnnotation(data, ownerObject)
```

Comment:
it looks like `ownerObject` only has the GVK set, does it need name/namespace set as well for the `SetOwnerAnnotation` to work properly?